packages/internal-helpers/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@astrojs/internal-helpers",
+  "description": "Internal helpers used by core Astro packages.",
+  "version": "0.1.0",
+  "type": "module",
+  "author": "withastro",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/withastro/astro.git",
+    "directory": "packages/internal-helpers"
+  },
+  "bugs": "https://github.com/withastro/astro/issues",
+  "exports": {
+    "./path": "./dist/path.js"
+  },
```

Comment:
Any particular reason why `types` and `main` fields are missing?

Reply:
This module is a bunch of subpackage helpers so there isn't a main module.